.github/workflows/correct_directory.yml

```
@@ -31,6 +31,7 @@ jobs:
                     '\nUnfortunately, the recipe was added directly in the `recipes` folder without its own subfolder.\n' +
                     'Please move the recipe file into a folder with the name of the package you want to submit.\n\n' +
                     'For example: if your recipe is currently under `recipes/<your_package>.yaml`, ' +
-                    'it should be moved to `recipes/<your_package>/meta.yaml`.\n' +
+                    'it should be moved to `recipes/<your_package>/meta.yaml` ' +
+                    'or, if you are using the new yaml based recipe format, `recipes/<your_package>/recipe.yaml`\n' +
```

I think we need a better label than "new yaml based recipe format" (this partially applies to `example-new-recipe` as well), especially since the old one is already called `meta.yaml`. I know it's not "pure" yaml, but to a casual user this is just confusing. Why not something like "v2 of the recipe specification" or something along those lines?

---

There has been a whiole discussion going on an we setteld on recipe v1. Ill change this.

```suggestion
                    'or, if you are using the recipe v1 format, `recipes/<your_package>/recipe.yaml`\n' +
```